db/migrate/20210428092832_create_patient_import_logs.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePatientImportLogs < ActiveRecord::Migration[5.2]
+  def change
+    create_table :patient_import_logs, id: :uuid do |t|
+      t.belongs_to :user, null: false, type: :uuid
+
+      t.uuid :record_id
+      t.string :record_type
```

Comment:
If I'm reading things right, this log will be basically be capturing:

```
user x imported patient Jane Doe
user x imported BP 1 for Jane Doe
user x imported rx drug for Jane Doe
```

and on and on for possibly thousands of records, correct?  

Is there a reason we can't send this data dog instead?  It looks like it would only be capturing UUIDs, a class name, and timestamps...do we really need to store this in the database for (basically) permeant storage?

Reply:
We can log these I think. Although datadog lets us search through only the last month.

Reply:
> We can log these I think. Although datadog lets us search through only the last month.

I like the sound of that ...I'm assuming a month of history would be fine, as this would be debugged right after running?

Reply:
We're only capturing these logs for audit compliance. As such, I've changed it up to re-use the existing `AuditLog`  mechanism we have in place. Imported records will be reported in the audit logs as an "import" by ~"import bot user"~ "admin who uploaded the spreadsheet"